README_zh.md

```diff
@@ -24,6 +24,25 @@ Golang 1.16+
 go get -u github.com/zc2638/swag@v1.4.3
 ```
 
+## 使用注意：
+time.time类型使用jsonTag来将其解析为string类型
+```go
+type XXX struct{
+  Now time.Time `json:",string"`
+}
+```
+
+不兼容如下解析：
+```go
+type XX struct {
+   Attributes1 []map[string]string //建议把map替换成struct
+   Attributes2 [][]string
+   Attributes3 map[string][][]string
+   Attributes4 map[string][]map[string]string //建议把map替换成struct
+   Attributes5 map[string]XX //不支持对结构体嵌套自身解析
+}
```

Comment:
Perhaps it would be better to describe a supporting structure here.

Reply:
It would be more intuitive to also display the incompatible versions for quick reference, making it even more evident by adding the compatible ones